.changeset/lazy-windows-tan.md

```diff
@@ -0,0 +1,5 @@
+---
+'xstate': patch
```

Comment:
technically this is a somewhat new feature, but i guess we treat minor versions as more grandiose ones than smth like this 😅

Reply:
It's a missing feature that should have been included 😜